### PR TITLE
WASM - Int64 global

### LINKED ISSUE #1996: WASM - Int64 imported global error
Make imported int64 globals throw at instantiation time instead of compile time

### PR DESCRIPTION
**Pull request by Cellule**
Fixes #1996

---
This change is [Reviewable](https://reviewable.io/reviews/microsoft/chakracore/1997)

### CODE REVIEW

File: lib/Runtime/Library/WebAssemblyModule.cpp [hunk 1]
```diff
@@ -154,9 +154,7 @@ WebAssemblyModule::EntryImports(RecyclableObject* function, CallInfo callInfo, .
     module->m_imports->Map([AddImport] (int index, Wasm::WasmImport * import) {
         AddImport(import, Wasm::ExternalKinds::Function);
     });
-    module->globals->Map([AddImport](int index, Wasm::WasmGlobal * global) {
-        AddImport(global->importVar, Wasm::ExternalKinds::Global);
-    });
+    // Todo:: Add globals (old implementation didn't work)
```
Comment (Cellule) [inline lib/Runtime/Library/WebAssemblyModule.cpp]: I already have a change pending that fixes this. I will open a PR after this is merged

**Comment by Cellule** [comment]
@dotnet-bot test Windows x86_debug please
@dotnet-bot test OSX osx_osx_debug_static please

**Comment by MikeHolman** [comment]
:shipit:

**Comment by Cellule** [comment]
The bug in x86_debug seems legit, I'll have to investigate more. Probably have to do with how we initialized globals in the environment

**Comment by Cellule** [comment]
It seems the bug has been there for a while. There was a break missing in the switch that sets the float global value. This caused to use the offset for float as a double and then set the unexpected memory.
I don't know why we didn't hit that bug before

**Comment by Cellule** [comment]
@dotnet-bot test OSX osx_osx_debug_static please

**Pull request merged**